src/main/java/mate/academy/lib/Injector.java

```diff
@@ -102,7 +102,7 @@ private Object createInstance(Class<?> clazz) {
         return newInstance;
     }
```

Comment:
This method is private and static, but it's not used anywhere in the class. Consider removing it.